CHANGELOG.md

```diff
@@ -0,0 +1,34 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+### Added
+
+- Specs
+- RuboCop linter and formatter
+- CI steps
+
+### Changed
+
+- Modularize CLI
+
+### Removed
+
+- Travis CI config
+
+## [2.6] - 2014-05-31
+
+[unreleased]: https://github.com/infinum/shaman
+[2.6]: https://rubygems.org/gems/shaman_cli/versions/2.6
```

Comment:
I'll need to add you to Rubygems and remove Štef so you'll be able to publish a release, remidn me tomorrow if I forget 🙏